src/gg20/keygen/api.rs

```diff
@@ -62,16 +62,16 @@ impl TryFrom<&[u8]> for SecretRecoveryKey {
 #[derive(Debug, Clone, Zeroize)]
 #[zeroize(drop)]
 pub struct PartyKeyPair {
-    pub(crate) ek: EncryptionKey,
-    pub(crate) dk: DecryptionKey,
+    pub(super) ek: EncryptionKey,
+    pub(super) dk: DecryptionKey,
 }
 
 #[derive(Debug, Clone)]
 pub struct PartyKeygenData {
-    pub(crate) encryption_keypair: PartyKeyPair,
-    pub(crate) encryption_keypair_proof: EncryptionKeyProof,
-    pub(crate) zk_setup: ZkSetup,
-    pub(crate) zk_setup_proof: ZkSetupProof,
+    pub(super) encryption_keypair: PartyKeyPair,
+    pub(super) encryption_keypair_proof: EncryptionKeyProof,
+    pub(super) zk_setup: ZkSetup,
+    pub(super) zk_setup_proof: ZkSetupProof,
```

Comment:
Hmm, I prefer to avoid polluting `mod.rs`. How about I remove the dependence on it in `r1.rs` by passing the fields directly to `r1::start`. No other files use this struct.

Reply:
It's needed in `secret_key_share.rs` too for `recover`, so I'm thinking of leaving it as is.